types/index.d.ts

```diff
@@ -87,6 +94,9 @@ declare namespace fastifyStatic {
     root: string | string[] | URL | URL[];
     prefix?: string;
     prefixAvoidTrailingSlash?: boolean;
+    hash?: boolean;
+    hashSkip?: string | string[];
+    hashPath?: string;
```

Comment:
Might be nice to only accept the params that configure hashing if hashing is on like so: 

```suggestion
    hash?: boolean | { skip: string | string[], path?: string };
```